tests/integration-tests/configs/common/common.yaml

```diff
@@ -394,16 +394,16 @@ scaling:
         oss: ["centos7"]
         schedulers: ["slurm"]
 schedulers:
-  test_awsbatch.py::test_awsbatch:
-    dimensions:
-      - regions: ["us-gov-west-1"]
-        instances: {{ common.INSTANCES_DEFAULT_X86 }}
-        oss: ["alinux2"]
-        schedulers: ["awsbatch"]
-      - regions: ["ap-northeast-1"]
-        instances: {{ common.INSTANCES_DEFAULT_ARM }}
-        oss: ["alinux2"]
-        schedulers: ["awsbatch"]
+#  test_awsbatch.py::test_awsbatch:
```

Comment:
let's add a comment in the file as well, to be clear why it's commented.

an alternative is to change the test code commenting the "mnp job submission" part but keeping the other